### PR TITLE
Update documentation with Azure-openai and ollama

### PR DESCRIPTION
**Pull request by galshubeli**
## Summary by CodeRabbit

- **New Features**
	- Added "Ollama" to the list of supported items.
	- Expanded support for LLM models to include Azure-OpenAI and Ollama with detailed use cases.

- **Documentation**
	- Updated `README.md` for improved clarity on supported LLMs and configuration requirements.
	- Added new environment variables for Azure integration: `AZURE_OPENAI_API_KEY`, `AZURE_ENDPOINT`, and `AZURE_API_VERSION`.
	- Specified that Ollama models are suitable only for the Q&A step after knowledge graph creation.
  
- **Chores**
	- Removed outdated `.env.example` file containing environment variable declarations. 
	- Introduced a new `.env.template` file with updated environment variable declarations and usage instructions. 

- **Style**
	- Corrected capitalization of "Google" for consistent formatting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,16 +47,26 @@ docker run -p 6379:6379 -p 3000:3000 -it --rm  -v ./data:/data falkordb/falkordb
 Currently, this SDK supports the following LLMs API:
 
 * [OpenAI](https://openai.com/index/openai-api) Recommended model:`gpt-4o`
-* [google](https://makersuite.google.com/app/apikey) Recommended model:`gemini-1.5-flash-001`
-
-Make sure that a `.env` file is present with all required credentials.
+* [Google](https://makersuite.google.com/app/apikey) Recommended model:`gemini-1.5-flash-001`
+* [Azure-OpenAI](https://ai.azure.com) Recommended model:`gpt-4o`
+* [Ollama](https://ollama.com/) Available only to the Q&A step. Recommended models: `llama3`. Ollama models are suitable for cases where the knowledge graph (KG) is predefined.
```
Comment (swilly22) [inline README.md:52]: What do you mean by KG is predefined ?

File: README.md [hunk 2]
```diff
@@ -47,16 +47,26 @@ docker run -p 6379:6379 -p 3000:3000 -it --rm  -v ./data:/data falkordb/falkordb
...
-* [google](https://makersuite.google.com/app/apikey) Recommended model:`gemini-1.5-flash-001`
-
-Make sure that a `.env` file is present with all required credentials.
+* [Google](https://makersuite.google.com/app/apikey) Recommended model:`gemini-1.5-flash-001`
+* [Azure-OpenAI](https://ai.azure.com) Recommended model:`gpt-4o`
+* [Ollama](https://ollama.com/) Available only to the Q&A step. Recommended models: `llama3`. Ollama models are suitable for cases where the knowledge graph (KG) is predefined.
+  
+    To get started with Ollama:
+    1. Install Ollama from https://ollama.com/
+    2. Pull your preferred model: `ollama pull llama3`
```
Comment (swilly22) [inline README.md:56]: Why do we have this section here?

If we believe this information is necessary, then let's create a dedicate section: `Getting Started` where each model is covered.

File: README.md [hunk 3]
```diff
@@ -47,16 +47,26 @@ docker run -p 6379:6379 -p 3000:3000 -it --rm  -v ./data:/data falkordb/falkordb
...
    GOOGLE_API_KEY="GOOGLE_API_KEY"
+   # API Key from Azure Portal
+   AZURE_OPENAI_API_KEY="AZURE_OPENAI_API_KEY"
+   # Endpoint format: https://{your-resource-name}.openai.azure.com
+   AZURE_ENDPOINT="AZURE_ENDPOINT"
+   # API Version (e.g., 2023-05-15)
+   AZURE_API_VERSION="AZURE_API_VERSION"
    ```
   
    </details>
```
Comment (swilly22) [inline README.md:72]: Replace this with a `.env.template` file, this is where users are expecting to find this information.

**Review by swilly22 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (galshubeli) [reply README.md:52]: Already created with other model

**Review by swilly22 — APPROVED** [review]

**Pull request merged**